opensearch/src/main/java/org/opensearch/sql/opensearch/executor/OpenSearchExecutionEngine.java

```diff
@@ -49,7 +49,8 @@ public void execute(PhysicalPlan physicalPlan, ExecutionContext context,
               result.add(plan.next());
             }
 
-            QueryResponse response = new QueryResponse(physicalPlan.schema(), result);
+            String rawResponse = physicalPlan.getRawResponse();
```

Comment:
If it is json format, result is not been used, only rawResponse will be used in response, right? Does it means the query plan should be different, if the query plan to fetch raw data?